packages/components/molecules/f-restaurant-card/src/components/RestaurantCard.v1.vue

```diff
@@ -49,6 +49,18 @@
             </component>
             <!-- END ERROR BOUNDARY -->
 
+            <!-- Local Legend label -->
+            <!-- START ERROR BOUNDARY -->
+            <component
+                :is="errorBoundary"
+                tier="3">
+                <div
+                    v-if="isPremier"
+                    :class="[$style['c-restaurantCard-localLegend']]">
+                    <legend-icon data-test-id="local-legend-icon" />
+                </div>
+            </component>
```

Comment:
I hate to say this, but the error boundary seems redundant here - if this was going to break, it would be during the process of building the component props object to pass down from the consumer (search, home etc) - which I hope we would handle with try/catches or some other form of graceful error handling. 

I think it might be worth dropping the error boundary from this component (can do the same for similar in future PRs to 'fix forward', reserving it for more complex components that have some internal logic that could be prone to breaking. 

What do you think?

Reply:
Totally Agree, I wasn't going to add it in, just saw other components low risk use it, so decided to just leave it in

Reply:
Feel free to question/challenge existing implementations - no need to copy if you think there's an issue with it! :)